lib/tasks/s3_file_conversion.rake

```diff
@@ -0,0 +1,39 @@
+require 'fileutils'
+include CarrierWave::MiniMagick
+
+namespace :s3 do
+  # Usage:
+  #     rake s3:file_conversion
+  desc "convert manually uploaded files to carrierwave objects"
+  task file_conversion: :environment do |task, args|
+    records = []
+
+    [Collection, Transcript].each do |klass|
+      klass.where.not(image_url: nil).find_each do |resource|
+        image_url = resource.read_attribute(:image_url)
+
+        if image_url.present? && resource.image.blank?
+            records << resource
+
+            dir_name = FileUtils.mkpath('tmp/s3_uploads/')
+            file_name = File.basename(image_url)
+            file_path = File.join(dir_name, file_name)
+
+            # Download the image from S3 and store in temp folder.
+            File.open(file_path, 'wb') do |file|
+              file.write open(image_url).read
+            end
+
+            # Upload the image from disk.
+            resource.vendor = Vendor.find_by uid: "voice_base"
```

Comment:
eh, if they're all meant to have the same vendor, then no big deal. you might want to store the vendor in a variable outside of the loop though, as it is, the `Vendor.find_by` is going to trigger a database query on each iteration of the loop